shell/src/main/java/org/apache/accumulo/shell/commands/ScanCommand.java

```diff
@@ -348,6 +387,9 @@ public Options getOptions() {
     optEndRowExclusive.setArgName("end-exclusive");
     scanOptRow = new Option("r", "row", true, "row to scan");
     scanOptColumns = new Option("c", "columns", true, "comma-separated columns");
```

Comment:
Could mention that -c is mutually exclusive here so users will avoid the thrown error above. One idea I had is if both options are passed in, we could just log a warning and then choose to ignore one of the arguments (either -c or -cf/cq). Thoughts on that?